certified-connectors/N-able Cloud Commander/apiProperties.json

```diff
@@ -0,0 +1,49 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "token": {
+        "type": "oauthSetting",
+        "oAuthSettings": {
+          "identityProvider": "oauth2",
+          "clientId": "[[DUMMY]]",
+          "scopes": [
+            "api://cloudapp.n-able.com/cloudapp_apis openid offline_access"
+          ],
+          "redirectMode": "GlobalPerConnector",
+          "redirectUrl": "https://global.consent.azure-apim.net/redirect/n-2dable-20cloud-20commander-5f3a2dd1faea314cd7-4e7247cbad6f0e7f",
+          "properties": {
+            "IsFirstParty": "False",
+            "IsOnbehalfofLoginSupported": false
+          },
+          "customParameters": {
+            "authorizationUrl": {
+              "value": "https://login.microsoftonline.com/common/oauth2/v2.0/authorize"
+            },
+            "tokenUrl": {
+              "value": "https://login.microsoftonline.com/common/oauth2/v2.0/token"
+            },
+            "refreshUrl": {
+              "value": "https://login.microsoftonline.com/common/oauth2/v2.0/token"
+            }
+          }
+        }
+      }
+    },
+    "iconBrandColor": "#000000",
+    "capabilities": [],
```

Comment:
Can you please confirm that the iconBrandColor is correct? 

Please refer this link https://learn.microsoft.com/en-us/connectors/custom-connectors/certification-submission#design-an-icon-for-your-connector

Reply:
Fixed